src/js/extensions/paste.js

```diff
@@ -176,11 +176,6 @@ var PasteHandler;
                     workEl.innerHTML = workEl.innerHTML.replace(/\n/gi, ' ');
 
                     switch (workEl.tagName.toLowerCase()) {
-                        case 'a':
```

Comment:
Why did you end up needing to remove this case?  And why did you need to move it to `pasteHTML()`?